client/client.c

```diff
@@ -77,6 +77,16 @@ TDNFPkgsToExclude(
         BAIL_ON_TDNF_ERROR(dwError);
     }
 
+    if (pTdnf->pConf->ppszExcludes)
+    {
+        printf("Warning: The following packages are excluded from tdnf.conf:\n");
```

Comment:
you might have to check if `quiet` flag is on before printing this

Reply:
Will fix this, thanks. Should we have a method for logging in TDNF (that takes pTdnf)? that can handle these nits instead of this Ad-Hoc prints? I can introduce one if the idea sounds okay.

Reply:
sounds good.

Reply:
Checked for quite in this instance. Will raise a separate PR for logging.